### PR TITLE
CodeBuilder: prolog code

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/ddmd/backend/cod3.c [hunk 1]
```diff
@@ -799,7 +796,6 @@ void outblkexitcode(block *bl, code* c, int& anyspill, const char* sflsave, symb
             }
             logexp(cdb,e,jcond,FLblock,(code *) bs1);
             nextb = bs2;
-            bl->Bcode = NULL;
```
Comment (yebblies) [inline src/ddmd/backend/cod3.c]: Won't this change be visible outside this function?  Why wouldn't we care about Bcode's value?

Comment (WalterBright) [inline src/ddmd/backend/cod3.c]: The only time `bl->Bcode` would be non-NULL when entering this function is if's a BCasm block. It took me a while to figure out that this invariant was the case. Leaving that assignment in there would just be confusing. It's also pointless, as after the function returns then `bl->Bcode` is set.

**Comment by WalterBright** [comment]
ping @yebblies

**Comment by wilzbach** [comment]
Closing and reopening to rekick Jenkins

**Pull request merged**